### DISCUSSION TITLE
How do I contribute my python graphing ..module?

### DISCUSSION THREAD

**Discussion by ssagerian** [AllskyTeam/allsky-modules#78] [General]
I would like to contribute a module to the cause, how do I submit it for  consideration?

**Comment by Alex-developer** [comment]
Best way is to for the repo and create a new branch with your module in it. then do a PR into the dev branch.

If you are unsure howto do that then just attach it as a zip file in here and I will sort it for you.

**Comment by Alex-developer** [comment]
Closing as no response from the user

**Comment by ssagerian** [comment]
Sorry, didnt realize you were waiting on me, yes, I can clone the repo and build something (soon).

**Comment by Alex-developer** [comment]
> Sorry, didnt realize you were waiting on me, yes, I can clone the repo and build something (soon).

No problem, just doing a bit of cleanup on discussions. I'll leave this open :-)
